evennia/commands/default/building.py

```diff
@@ -718,9 +720,10 @@ def func(self):
         caller = self.caller
 
         if not self.lhs:
-            string = "Usage: @dig[/teleport] roomname[;alias;alias...][:parent] [= exit_there"
+            string = "Usage: @dig[/teleport] <roomname>[;alias;alias...][" \
```

Comment:
Whereas leaving the trailing `[` on the line above is a good idea to delimit Python code, I feel it might be confusing the eye when doing it for Command-syntax like this. I much prefer the way you do it in the `@dig` version above, when including the full `[ ... ]` on the line, but indenting the next line. Makes it easier to see what belongs to which without having to resort to counting brackets.